\doxysection{urllib3.\+exceptions.\+Timeout\+Error Class Reference}
\hypertarget{classurllib3_1_1exceptions_1_1_timeout_error}{}\label{classurllib3_1_1exceptions_1_1_timeout_error}\index{urllib3.exceptions.TimeoutError@{urllib3.exceptions.TimeoutError}}
Inheritance diagram for urllib3.\+exceptions.\+Timeout\+Error\+:\begin{figure}[H]
\begin{center}
\leavevmode
\includegraphics[height=6.000000cm]{classurllib3_1_1exceptions_1_1_timeout_error}
\end{center}
\end{figure}


\doxysubsection{Detailed Description}
\begin{DoxyVerb}Raised when a socket timeout error occurs.

Catching this error will catch both :exc:`ReadTimeoutErrors
<ReadTimeoutError>` and :exc:`ConnectTimeoutErrors <ConnectTimeoutError>`.
\end{DoxyVerb}
 

The documentation for this class was generated from the following file\+:\begin{DoxyCompactItemize}
\item 
C\+:/\+Users/kmarty/\+Documents/\+Labo3\+\_\+architecture\+\_\+killian\+\_\+marty/\+Lib/site-\/packages/urllib3/exceptions.\+py\end{DoxyCompactItemize}
